frontend/src/components/scheduleEntry/ScheduleEntries.vue

```diff
@@ -4,16 +4,14 @@
       :scheduleEntries="scheduleEntries"
       :loading="apiScheduleEntries._meta.loading"
       :showActivityCreateDialog="showActivityCreateDialog"
-      :showActivityEditDialog="showActivityEditDialog" />
+      :on="eventHandlers" />
```

Comment:
Ich verstehe nicht ganz warum all diese Änderungen in dieser Datei nötig sind um den Picasso mit Composition API zu implementieren... Aber es sieht ein Stück weit sauberer aus, also wenns funktioniert okay

Reply:
Das hat hauptsächlich damit zu tun, das ich nun auf Vue-Event höre, anstatt callbacks über die Props nach unten zu geben - womit ich soeben bemerkt habe, dass ich oben das showActivitiyCreateDialog-Prop vergessen habe zu entfernen :-)

Eventuell kann ich's nochmals etwas vereinfachen wenn ich click / open aus drag & drop entferne und separat implementiere.

Viel der Logik in diesem File könnte eigentlich auch in der Picasso-Komponenten drin sein. Ich glaube ScheduleEntries gibt es historisch, weil man früher mal zwischen Picasso und Listenansicht switchen konnte und ScheduleEntries war das gemeinsame Dach darüber. Die Listenansicht gibt es aber gar nicht mehr. Eigentlich könnte man damit ScheduleEntries und Picasso zu einer Komponente verschmelzen. Das würde etwas von der Komplexität wegnehmen.

@manuelmeister Any comments to this proposal?

Reply:
@usu Ich hätte mir eigentlich schon gedacht, dass eine Listenansicht wieder nötig ist.

Reply:
Ok, dann lass ich die Aufteilung zwischen `Picasso.vue` und `ScheduleEntries.vue` vorerst mal so wie's jetzt ist.